cmd/crc/cmd/start.go

```diff
@@ -63,8 +63,13 @@ func runStart(arguments []string) (*machine.StartResult, error) {
 
 	checkIfNewVersionAvailable(config.Get(cmdConfig.DisableUpdateCheck).AsBool())
 
-	if err := preflight.StartPreflightChecks(config); err != nil {
-		return nil, err
+	// If skip-all-preflight-checks is true then don't run the prefligh checks
+	if !config.Get(cmdConfig.SkipAllPreflightChecks).AsBool() {
+		if err := preflight.StartPreflightChecks(config); err != nil {
+			return nil, err
+		}
+	} else {
+		logging.Info("All preflight checks are skipped")
```

Comment:
can this be inverted ?

```
if config.Get(cmdConfig.SkipAllPreflightChecks).AsBool() {
    logging.Info("All preflight checks are skipped")
} else {
    ...
}
```

Why the inversion of the value? Having the output clearly visible explains the purpose of this.

Reply:
Changed!